packages/monorepo/src/components/nx-configurator.ts

```diff
@@ -537,12 +537,9 @@ export class NxConfigurator extends Component implements INxProjectCore {
    */
   private _addLicenses() {
     [this.project, ...this.project.subprojects]
-      .filter(
-        (p) =>
-          !this.licenseOptions?.disableDefaultLicenses &&
-          p.components.find((c) => c instanceof License) === undefined
-      )
+      .filter(() => !this.licenseOptions?.disableDefaultLicenses)
       .forEach((p) => {
+        p.tryRemoveFile("LICENSE");
```

Comment:
I don't think you need to consider case variations (like "License" or "license"), but GitHub license files can be LICENSE or LICENSE.md

Reply:
Not necessary as we are only dealing with LICENSE files here.